src/ios/NewRelicCordovaPlugin.m

```diff
@@ -3,7 +3,7 @@
 //  See:
 //    https://docs.newrelic.com/docs/releases/ios for release notes
 //
-//  Copyright (c) 2017 New Relic. All rights reserved.
+//  Copyright (c) 2017 - Present New Relic. All rights reserved.
 //  See https://docs.newrelic.com/docs/licenses/ios-agent-licenses for license details
```

Comment:
@ywang-nr Sorry for the late feedback, but is this file still current? And why just one for iOS?

Reply:
Let me double check with IOS team.